samples/matter/common/src/bridge/ble_connectivity_manager.h

```diff
@@ -32,6 +32,23 @@ class BLEConnectivityManager {
 	static constexpr uint16_t kMaxConnectedDevices = CONFIG_BT_MAX_CONN - 1;
 	static constexpr uint8_t kMaxServiceUuids = CONFIG_BT_SCAN_UUID_CNT;
 
+	/**
+	 * States for indication the most important BLE Connectivity Manager states.
+	 * If the new state occurs the previous one is automatically disabled until the next occurrence.
```

Comment:
This is an odd sentence. I'm guessing it tries to say something like "Only the latest state is indicated" or "A new state disables the previous state"?